hc_dpki/Cargo.toml

```diff
@@ -9,7 +9,8 @@ lazy_static = "~1.2.0"
 base64 = "0.10.0"
 holochain_core_types = { path = "../core_types" }
 holochain_sodium = { path = "../sodium" }
-serde = "1.0"
+serde = "1.0.87"
```

Comment:
Maybe we should update to the same version of serde everywhere so there aren't lots of extra versions compiled into the code?

Reply:
Reverted